src/bvals/comms/bnd_info.cpp

```diff
@@ -277,7 +278,7 @@ BndInfo::BndInfo(MeshBlock *pmb, const NeighborBlock &nb,
   allocated = v->IsAllocated();
   alloc_status = v->GetAllocationStatus();
 
-  buf = combuf->buffer();
+  if (combuf != nullptr) buf = combuf->buffer();
```

Comment:
This was a little bit of a hack so that I could build a `BndInfo` object without a comm buffer and use the `BndInfo` to get the exact size of the buffer (then throw it away). Definitely should be cleaned up though.

Reply:
(see line 322 below)

Reply:
Can you add a comment to this effect? Otherwise its a bit confusing.

Reply:
sure, done.